google-cloud-clients/google-cloud-speech/samples/src/main/java/com/google/cloud/examples/speech/v1/SpeechTranscribeAsync.java

```diff
@@ -0,0 +1,118 @@
+
+package com.google.cloud.examples.speech.v1;
+
+import com.google.api.gax.longrunning.OperationFuture;
+import com.google.cloud.speech.v1.LongRunningRecognizeMetadata;
+import com.google.cloud.speech.v1.LongRunningRecognizeRequest;
+import com.google.cloud.speech.v1.LongRunningRecognizeResponse;
+import com.google.cloud.speech.v1.RecognitionAudio;
+import com.google.cloud.speech.v1.RecognitionConfig;
+import com.google.cloud.speech.v1.SpeechClient;
+import com.google.cloud.speech.v1.SpeechRecognitionAlternative;
+import com.google.cloud.speech.v1.SpeechRecognitionResult;
+import com.google.protobuf.ByteString;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import org.apache.commons.cli.CommandLine;
+import org.apache.commons.cli.DefaultParser;
+import org.apache.commons.cli.Option;
+import org.apache.commons.cli.Options;
+
+public class SpeechTranscribeAsync {
+  // [START speech_transcribe_async]
+  /*
+   * Please include the following imports to run this sample.
+   *
+   * import com.google.api.gax.longrunning.OperationFuture;
+   * import com.google.cloud.speech.v1.LongRunningRecognizeMetadata;
+   * import com.google.cloud.speech.v1.LongRunningRecognizeRequest;
+   * import com.google.cloud.speech.v1.LongRunningRecognizeResponse;
+   * import com.google.cloud.speech.v1.RecognitionAudio;
+   * import com.google.cloud.speech.v1.RecognitionConfig;
+   * import com.google.cloud.speech.v1.SpeechClient;
+   * import com.google.cloud.speech.v1.SpeechRecognitionAlternative;
+   * import com.google.cloud.speech.v1.SpeechRecognitionResult;
+   * import com.google.protobuf.ByteString;
+   * import java.nio.file.Files;
+   * import java.nio.file.Path;
+   * import java.nio.file.Paths;
+   */
+
+  /**
+   * Transcribe a long audio file using asynchronous speech recognition
+   *
+   * @param localFilePath Path to local audio file, e.g. /path/audio.wav
+   */
+  public static void sampleLongRunningRecognize(String localFilePath) {
+    try (SpeechClient speechClient = SpeechClient.create()) {
+      // localFilePath = "resources/brooklyn_bridge.raw";
+
+      // The language of the supplied audio
+      String languageCode = "en-US";
+
+      // Sample rate in Hertz of the audio data sent
+      int sampleRateHertz = 16000;
+
+      // Encoding of audio data sent. This sample sets this explicitly.
+      // This field is optional for FLAC and WAV audio formats.
+      RecognitionConfig.AudioEncoding encoding = RecognitionConfig.AudioEncoding.LINEAR16;
+      RecognitionConfig config =
+          RecognitionConfig.newBuilder()
+              .setLanguageCode(languageCode)
+              .setSampleRateHertz(sampleRateHertz)
+              .setEncoding(encoding)
+              .build();
+      Path path = Paths.get(localFilePath);
+      byte[] data = Files.readAllBytes(path);
+      ByteString content = ByteString.copyFrom(data);
```

Comment:
a comment & blank lines talking about reading from a local file (you'll need to adapt for your needs) would be good here.